CMakeLists.txt

```diff
@@ -227,7 +227,9 @@ set(AER_LIBRARIES
 	Threads::Threads
 	CONAN_PKG::spdlog
 	${DL_LIB}
-	${THRUST_DEPENDANT_LIBS})
+	${THRUST_DEPENDANT_LIBS}
+	CONAN_PKG::sundials
+	CONAN_PKG::boost)
```

Comment:
:'(

Is it possible to only get the ODEINT part of boost?

Reply:
We are using boost as a header only lib so we are only using what we include.